includes/managers/widgets.php

```diff
@@ -3,7 +3,6 @@
 
 use Elementor\Core\Common\Modules\Ajax\Module as Ajax;
 use Elementor\Core\Utils\Collection;
-use Elementor\Core\Utils\Exceptions;
```

Comment:
Is this intentional / needed?

Reply:
<img width="646" alt="image" src="https://github.com/elementor/elementor/assets/164857959/5a9b091e-1313-4478-a7f3-06a22cb9917e">
sorry but I haven't this import on my side, do you check my last commit?

Reply:
It's good that it was removed because it was unused :)

Reply:
fixed